lib/flop/schema.ex

```diff
@@ -249,7 +249,7 @@ defprotocol Flop.Schema do
 
       Owner
       |> join(:left, [o], p in assoc(o, :pets), as: :pets)
-      |> preload([pets: p], [pets: p])
+      |> preload([pets: p])
```

Comment:
No, the first one is the named binding for `:pets`, and the second one preloads pets from that binding. In your version, the preload is done with a separate query.